src/README.md

```diff
@@ -1,31 +1,65 @@
-# OpenThread on EFR32MG12 Example
+# OpenThread on EFR32
+
+##### Table of Contents
+
+- [Prerequisites](#prerequisites)
+  - [Toolchain](#toolchain)
+  - [Flashing and debugging tools](#flash-debug)
+- [Building examples](#build)
+- [Flashing binaries](#flash)
+  - [Simplicity Commander](#simplicity-commander)
+  - [J-Link GDB Server](#jlink-gdb-server)
+  - [J-Link Commander](#jlink-commander)
+- [Run the example with EFR32 boards](#example)
+- [Additional features](#additional-features)
+- [Verification](#verification)
+
+---
 
 This directory contains example platform drivers for the [Silicon Labs EFR32MG12][efr32mg] based on [EFR32™ Mighty Gecko Wireless Starter Kit][slwstk6000b] or [Thunderboard™ Sense 2 Sensor-to-Cloud Advanced IoT Development Kit][sltb004a].
```

Comment:
[Silicon Labs EFR32MG12] should be [Silicon Labs EFR32MG]

Notice the removal of "12". Please search the rest of this file and make sure similar references have been updated.